python/src/equistore/tensor.py

```diff
@@ -1,12 +1,22 @@
 import copy
 import ctypes
+import os
+import pickle
+import sys
+import tempfile
+
+
+if (sys.version_info.major >= 3) and (sys.version_info.minor >= 8):
+    from pickle import PickleBuffer
```

Comment:
Could you move this import below the `from typing ...`. This keeps import grouped in stdlib/external/internal

Reply:
moved